src/styles/Terminal/Terminal.scss

```diff
@@ -1,5 +1,16 @@
 .terminal {
   justify-content: center;
   margin: auto;
+  position: relative;
   width: 80%;
 }
+
+.copybutton {
+  position: absolute;
+  right: 14px;
+  top: 14px;
+
+  &:hover {
```

Comment:
Can you make the hover state also set the cursor to be a pointer? Otherwise, looks good!

Reply:
https://github.com/uclaacm/parcel-pointers/assets/104863284/ab7d1e01-b9b1-4d58-8e01-5a3e14d085b8